### DISCUSSION TITLE
PCB Design

### DISCUSSION THREAD

**Discussion by ChrisLizon** [Dilbert66/esphome-vistaECP#131] [General]
Does anyone have any examples of PCBs they have made up for the Vista ESP interface? i know @Dilbert66 has some examples for his DSC but I would like to do one for my Vista panel with an W32-Eth1. I have taken a crack at it in EasyEDA, but I'm hoping someone can "proofread" my PCB before I send it off to have them made. 
[Vista_W32-Eth01.zip](https://github.com/Dilbert66/esphome-vistaECP/files/15456504/Vista_W32-Eth01.zip)

**Comment by monzsca** [comment]
Maybe too late, but I made. I used Kicad. It's for use with an Inland ESP32 (Microcenter) board. I picked the wrong footprint for the ESP32 board, had to bend pins to make it fit. I didn't bother with silkscreen etc since I etched the board myself.

On yours, it looks like the WT32-ETH01 boards don't have reset and boot buttons. You may want to add those or headers to access them. Same with RX and TX for initial programming or if an OTA update fails and needs to be manually loaded. I'd make the traces a bit wider and the ones going between ESP32 pins could go around the ESP32.

[vistaesphome.zip](https://github.com/user-attachments/files/15917871/vistaesphome.zip)
